### PR TITLE
feat(inactive-page): style inactive page

### PR DESCRIPTION
**Pull request by lmorchard**
Let's try this PR again

### CODE REVIEW

**Comment by johngruen** [comment]
@pdehaan this is what i'm talking about...I'll merge once it passes!

thanks @lmorchard!

File: idea_town/frontend/static-src/app/templates/account-inactive-page.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
-      <header data-hook="main-header"></header>
-      <h1>Account Inactive</h1>
-      <h2>We're not quite ready for you yet!<br>
-          Please come back in a few weeks.</h2>
+  <div class="page" id="full-page-notice">
+    <section class="main-content" data-hook="account-inactive-page">
+      <div class="vertical-centering-container">
+        <div id="beta-notice-modal" class="delayed-fade-in">
+          <h1>Thanks for signing up!</h1>
+          <h2>We'll let you know as soon as Idea Town is ready.</h2>
```
Comment (pdehaan) [inline idea_town/frontend/static-src/app/templates/account-inactive-page.js:7]: What does _"We'll let you know"_ mean? Are we collecting email addresses somewhere?
How will we notify these random people?

**Comment by johngruen** [comment]
@pdehaan yerp

File: idea_town/frontend/static-src/app/templates/account-inactive-page.js [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+          <div class="modal-controls">
+            <a href="/"><button class="button primary">Home</button></a>
```
Comment (pdehaan) [inline idea_town/frontend/static-src/app/templates/account-inactive-page.js:9]: I spotted this in #267, but https://html5.validator.nu/ doesn't seem to like nesting `<button>` tags in an `<a>` element for unexplained reasons.

> **Error:** The element `button` must not appear as a descendant of the `a` element.
> From line 15, column 25; to line 15, column 55
> 
> ``` html
> href="/"><button class="button primary">Home</
> ```

I tried shuffling some elements and classes in the dev tools locally, but all experiments looked pretty tragic, so we may need to adjust some CSS styling as well (in a future PR).

File: idea_town/frontend/static-src/app/templates/account-inactive-page.js (hunk 1 above)
Comment (lmorchard) [inline idea_town/frontend/static-src/app/templates/account-inactive-page.js:7]: We collect the email address passed along with the FxA sign-in, as part of an invitation mode system introduced here:

https://github.com/mozilla/idea-town/issues/128#issuecomment-148826004
https://github.com/mozilla/idea-town/commit/f1d09e9cb35d3c1e5b0df39aa31249d13bd61074

We don't actually have the machinery to notify these users via email yet, though:

https://github.com/mozilla/idea-town/issues/129

**Comment by johngruen** [comment]
@pdehaan i'll fix these nits in my 404 page pr which is incoming soon

**Comment by johngruen** [comment]
@pdehaan which is to say, i'll make that string explicitly mention the email

**Comment by pdehaan** [comment]
@johngruen

**Pull request merged**